sector-builder/src/disk_backed_storage.rs

```diff
@@ -11,17 +11,29 @@ use crate::error::SectorManagerErr;
 use crate::store::{ProofsConfig, SectorConfig, SectorManager, SectorStore};
 use crate::util;
 
-// These sizes are for SEALED sectors. They are used to calculate the values of setup parameters.
-// They can be overridden by setting the corresponding environment variable (with FILECOIN_PROOFS_ prefix),
-// but this is not recommended, since some sealed sector sizes are invalid. If you must set this manually,
-// ensure the chosen sector size is a multiple of 32.
-
 pub struct DiskManager {
     staging_path: String,
     sealed_path: String,
 }
 
+impl DiskManager {
+    fn sector_path<P: AsRef<Path>>(&self, sector_dir: P, access: &str) -> PathBuf {
```

Comment:
make this a standalone function, it doesn't use `self`

Reply:
done